pkgs/development/python-modules/python3-msrplib/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
```

`stdenv` is not used.

```suggestion
{ lib
```